deploy/charts/cert-manager/README.template.md

```diff
@@ -19,16 +19,9 @@ Before installing the chart, you must first install the cert-manager CustomResou
 This is performed in a separate step to allow you to easily uninstall and reinstall cert-manager without deleting your installed custom resources.
 
 ```bash
-# Kubernetes 1.15+
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/{{RELEASE_VERSION}}/cert-manager.crds.yaml
-
-# Kubernetes <1.15
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/{{RELEASE_VERSION}}/cert-manager-legacy.crds.yaml
+$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/{{RELEASE_VERSION}}/cert-manager.crds.yaml
```

Comment:
I notice there's still a 1.15 specific section below regarding the deletion of cert-manager.

Are you leaving that in place for users who might be upgrading?

Reply:
Just an oversight  here

Reply:
For deletes the legacy vs normal actually doesn't matter